src/widgets/configure/configure.js

```diff
@@ -0,0 +1,50 @@
+import isPlainObject from 'lodash/isPlainObject';
+const usage = `Usage:
+search.addWidget(
+  instantsearch.widgets.configure({
+    // any searchParameter
+  })
+);
+Full documentation available at https://community.algolia.com/instantsearch.js/widgets/configure.html
```

Comment:
The URL is incorrect, it should be https://community.algolia.com/instantsearch.js/v2/widgets/configure.html